test/torchaudio_unittest/prototype/hdemucs_test_impl.py

```diff
@@ -106,3 +116,88 @@ def test_decoder_output_shape_time(self):
 
         assert z.size() == (batch_size, chout, t * stride)
         assert y.size() == (batch_size, chin, t)
+
+
+@skipIfNoModule("demucs")
+class CompareHDemucsOriginal(TorchaudioTestCase):
+    """Test the process of importing the models from demucs.
+
+    Test methods in this test suite will check to
+    1. Assure correctness in factory functions, comparing with original hybrid demucs
+    """
+
+    def _get_original_model(self, sources: List[str], nfft: int, depth: int):
+        from demucs import hdemucs as original
+
+        original = original.HDemucs(sources, nfft=nfft, depth=depth)
+        return original
+
+    @parameterized.expand(
+        [
+            (["bass", "drums", "other", "vocals"],),
```

Comment:
this can also be moved into a single function/var (see [example](https://github.com/pytorch/audio/blob/main/test/torchaudio_unittest/models/wav2vec2/fairseq_integration_test.py#L44)), since several tests reuse it